web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -369,8 +369,13 @@ import {get_string_diff} from "../../src/util";
         }
         return i[0]
       })
+      if(this.query.length!==0){
```

Comment:
This will change how all typeaheads work, not just search. Are we okay with making that change throughout? 

If so, it's also recommended to make this change a separate commit so that it's easier to find and revert if typeaheads elsewhere are having problems.

Reply:
And this would actually be two typeahead-changes commits. It seems like the first change is to remove the highlight on the first element when the search string is empty. The second separate change is to do nothing on `enter` when the search string is empty. Both of these for all instances of typeaheads.

Reply:
@evykassirer , it would change how all typeaheads work, but I guess ```helpOnEmptyStrings = true``` would display helper typeahead on empty string only for the top query search bar, whereas for all other searches... they dont display this helper typeahead when empty string.... So I guess none of the other should be affected by it.